### ISSUE TITLE
Delete of org with ID 1

### ISSUE THREAD

**Issue by FStefanni** [grafana/grafana#30016]
**What happened**:
Hi,

I have incurred in a possible bug.

Running Grafana 7.2.0 as docker container:
1. Created two new orgs
2. Delete the default "Main Org."
3. Restart the container: the container did not start, reporting some errors
4. So I connected to the grafana.db, and manually recreated the entry in org table, setting it as ID 1 and name Main Org.
5. Restart the container: it works

**What you expected to happen**:

We should be able to freely delete orgs, including the org with ID 1 (the default Main Org), and grafana should continue to work.
Or, if this is not possible, Grafana should ensure that the org with ID 1 should not be deletable via gui.

*NOTE*: I suppose the issue is the ID 1 since i have other instances, where the Main org has been renamed, and they work fine.
So the issue should be the ID and not the name of the org.

**How to reproduce it (as minimally and precisely as possible)**:

See above.

**Anything else we need to know?**:
No

**Environment**:
- Grafana version: 7.2.0
- OS Grafana is installed on: official docker image running on ubuntu 18.04

Regards

**Comment by wbrowne** [comment]
@FStefanni Thanks for logging this issue.

>the container did not start, reporting some errors

Could you provide more detail here? It would be a great help if you could share the logs.

Also, can you confirm if you're running the  `grafana/grafana-7.2.0-ubuntu` image?

**Comment by FStefanni** [comment]
Hi,

1. Sorry, I was not clear. The image is the official Alpine based image of Grafana 7.2.0. The host on which the container runs is Ubuntu 12.04. The image has been customized by installing the following plugins:
    * grafana-multibar-graph-panel
    * btplc-status-dot-panel
    * grafana-clock-panel
    * michaeldmoore-multistat-panel
    * grafana-worldmap-panel
    * a custom plugin by me
2. Sorry, but it was in production, so I fixed the issue as fast as possible, and only after I thought to report the issue, so I have no  logs. But probably you can try to reproduce the error

Regards.

**Comment by ying-jeanne** [comment]
Hi @FStefanni, 

Thanks for providing further information,

I tried 2 approaches to reproduce the issue:
1. running Grafana docker image grafana/grafana:7.2.0, following the steps to create 2 org, delete the default org with id 1, but was not able to reproduce the issue
2. running ubuntu docker image, then install Grafana in docker, following the steps, but still not able to reproduce the issue.

Also, do the plugins were installed by setting the env variable (GF_INSTALL_PLUGINS) or by modifying the Grafana docker image? It might have an impact as well.

For now, I would have to close the ticket, let us know if the problem resists, some logs would be a great help.

Regards,

**Comment by FStefanni** [comment]
Hi,

the plugins were installed by creating a custom image, and placing inside the Dockefile commands as:
```
RUN grafana-cli --pluginsDir "$GF_PATHS_PLUGINS" plugins install <plugin name>
```
and, obviously, setting the `GF_PATHS_PLUGINS` env var.
I have not tried to reproduce the error, so I am not able to give you further help at the moment.
But if ti will happen in the future, I'll open a new issue for sure (hopefully with some logs).

Regards
